jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1226,24 +1236,38 @@ def _conv_general_dilated(lhs, rhs, window_strides, padding, lhs_dilation,
 tf_impl_with_avals[lax.conv_general_dilated_p] = _conv_general_dilated
 
 
-def _dot_general(lhs, rhs, dimension_numbers, precision, preferred_element_type):
+def _dot_general(lhs, rhs, *,
```

Comment:
The third argument is surprising to me.

Reply:
The `*`? It says that the rest of the arguments must be passed as kwargs, not positional.